core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -596,16 +596,17 @@ private static ElevationProvider createElevationProvider(GraphHopperConfig ghCon
         } else if (eleProviderStr.equalsIgnoreCase("srtmgl1")) {
             elevationProvider = new SRTMGL1Provider(cacheDirStr);
         } else if (eleProviderStr.equalsIgnoreCase("multi")) {
-            elevationProvider = new MultiSourceElevationProvider(cacheDirStr);
+            MultiSourceElevationProvider provider = new MultiSourceElevationProvider(cacheDirStr);
+            provider
+                    .setAutoRemoveTemporaryFiles(removeTempElevationFiles)
+                    .setInterpolate(interpolate)
+                    .setDAType(elevationDAType);
+            if (!baseURL.isEmpty())
+                provider.setBaseURL(baseURL);
```

Comment:
So far we called `setAutoRemoveTemporaryFiles`, `setInterpolate`, `setDAType` and `setBaseURL` for all the different elevation providers, but now you call these methods only for the `MultiSourceElevationProvider`? Is this intended, and why?

Reply:
@easbar, I addressed your requested changes. Is the PR good for you now? Let me know so that we can move forward.